web/public/locales/da.json

```
@@ -113,8 +113,10 @@
     "maintitle": "Realtids-CO₂-udledning fra elforbruget",
     "legend": "Forklaring",
     "oops": "Hov! Vi kan ikke få kontakt til serverne. Vi prøver igen om et par sekunder.",
-    "newversion": "En ny version er tilgængelig! Tryk <a onClick=\"location.reload(true);\">her</a> for at opdatere.",
-    "retrynow": "Prøv igen nu"
+    "newversion": "En ny version er tilgængelig!",
+    "retrynow": "Prøv igen nu",
+    "reload": "Genindlæs",
+    "dismiss": "Afskedige"
```

One tiny nitpick as a native Danish speaker :)
```suggestion
    "dismiss": "Annuller"
```